README.md

```diff
@@ -33,6 +33,15 @@ To execute, run:
 
 To develop this project, use your favorite text editor, or an integrated development environment with Python support, such as [PyCharm](https://www.jetbrains.com/pycharm/).
 
+### Installation
+
+Install in editable mode and with extra developer dependencies:
+
+    pip install -e .[dev]
```

Comment:
In documentation, I prefer to see the long form command-line options (`--editable`). It feels more "documenty" that way.

Reply:
done!